codalab/worker/worker.py

```diff
@@ -622,6 +625,15 @@ def initialize_run(self, bundle, resources):
                 bundle=bundle,
                 bundle_path=os.path.realpath(bundle_path),
                 bundle_dir_wait_num_tries=Worker.BUNDLE_DIR_WAIT_NUM_TRIES,
+                bundle_profile_stats={
+                    RunStage.PREPARING: self.initstats(),
+                    RunStage.RUNNING: self.initstats(),
+                    RunStage.CLEANING_UP: self.initstats(),
+                    RunStage.UPLOADING_RESULTS: self.initstats(),
+                    RunStage.FINALIZING: self.initstats(),
+                    RunStage.FINISHED: self.initstats(),
```

Comment:
Should FINISHED (and RESTAGED) be actual stages where we have stats for?  It seems like it's more of an endpoint.

Reply:
Yeah, good catch. I just initially included all the stages for quick iteration.